### PR TITLE
make callbackery work better on mobile

### PR DESCRIPTION
**Pull request by oetiker**
* collaps navar buttons into menu
* move labels over form fields

### CODE REVIEW

File: lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Action.js [hunk 1]
```diff
@@ -173,14 +187,16 @@ qx.Class.define("callbackery.ui.plugin.Action", {
...
                     case 'download':
                     case 'display':
+                        mmButton = new qx.ui.menu.Button(label);
                         if (btCfg.addToMenu != null) {
                             button = new qx.ui.menu.Button(label);
                         }
                         else {
                             button = new buttonClass(label);
-                        }
+                                                }
```
Comment (zaucker) [inline lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Action.js:196]: indentation

File: lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Action.js [hunk 2]
```diff
@@ -173,14 +187,16 @@ qx.Class.define("callbackery.ui.plugin.Action", {
...
                         if (btCfg.key){
                             this._buttonMap[btCfg.key]=button;
+                            this._mmButtonMap[btCfg.key]=mmButton;
```
Comment (zaucker) [inline lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Action.js:199]: s/=/ = /

File: lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Form.js [hunk 3]
```diff
@@ -125,7 +125,7 @@ qx.Class.define("callbackery.ui.plugin.Form", {
                 },
                 this
             );
-
+
```
Comment (zaucker) [inline lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Form.js:128]: Trailing spaces

File: lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Table.js [hunk 4]
```diff
@@ -51,11 +51,17 @@ qx.Class.define('callbackery.ui.plugin.Table', {
...
+            toolbar.setOverflowIndicator(mobileMenu);
+            toolbar.setOverflowHandling(true);
+
             toolbar.add(action);
+            toolbar.setRemovePriority(action,1);
             toolbar.addSpacer();
             var form = this._form = new callbackery.ui.form.Auto(cfg.form,null,callbackery.ui.form.renderer.HBox, this);
             toolbar.add(form);
-	    return toolbar;
+	        return toolbar;
```
Comment (zaucker) [inline lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Table.js:64]: Indentation

**Review by zaucker — CHANGES_REQUESTED** [review]
Some cosmetical changes

**Review by zaucker — APPROVED** [review]

**Review by zaucker — APPROVED** [review]

File: CHANGES [hunk 5]
```diff
@@ -1,3 +1,22 @@
+0.47.3 2023-08-29 16:20:30 +0200 Tobias Oetiker <tobi@oetiker.ch>
+
+ - make table columns with flex with work on mobile devices
```
Comment (maettu) [inline CHANGES:3]: I think the second "with" should be removed.

**Pull request merged**